instructions.md

```diff
@@ -121,8 +121,25 @@ repo_url: https://github.com/yoyo.dyne/turbo-encabulator
+| 7   | title of product submission contains Kubernetes release version and product name | the submission title is missing either a Kubernetes release version (v1.xx) or product name                                                                                                                                                       | Not verifiable |      | Checking the PR title format                                                                                                       |
+| 8   | the e2e.log output contains the Kubernetes release version                       | it seems the e2e.log does not contain the Kubernetes release version that match the submission title                                                                                                                                              | Not verifiable |      | Matching the e2e.log release version with the PR title release version                                                             |
+| 9   | the submission release version is a supported version of Kubernetes              | the Kubenetes release version in this pull request does not qualify for conformance submission anymore (https://github.com/cncf/k8s-conformance/blob/master/terms-conditions/Certified_Kubernetes_Terms.md#qualifying-offerings-and-self-testing) | Not verifiable |      | Only the current release version and two version prior is supported for conformance certification                                  |
+| 10  | all required conformance tests in the junit_01.xml are present                   | it appears that some tests are missing from the product submission                                                                                                                                                                                | Not verifiable |      |                                                                                                                                    |
+| 11  | all tests pass in e2e.log                                                        | it appears that some tests failed in the product submission                                                                                                                                                                                       | Not verifiable |      |                                                                                                                                    |
+| 12  | the tests in junit_01.xml and e2e.log match                                      | it appears that there is a mismatch of tests in junit_01.xml and e2e.log                                                                                                                                                                          | Not verifiable |      | Verify that the junit_01.xml file and e2e.log file contain the same tests                                                          |
 
```

Comment:
| Final check  |  All requirements 12 have passed for the submission!    |                                                                                                                                                                         | Not verifiable | release-"+s.KubernetesReleaseVersion
release-documents-checked    | The step will add the release+version &  release-documents-checked labels once all 12 check have passed                               |